assemblies/assembly_managing-business-central/main.adoc

```diff
@@ -0,0 +1,91 @@
+[id='assembly_managing-business-central']
+
+include::_artifacts/document-attributes.adoc[]
+
+= Managing {CENTRAL}
+// Context attribute is assembly specific and enables module reuse between assemblies.
+:doctype: book
+:imagesdir: _images
+:context: managing-business-central
+include::_artifacts/author-group.adoc[]
+
+// Purpose statement for the assembly
+As an administrator, you can customize {CENTRAL} using the *Admin Settings*. You can use the *Admin Settings* to customize the following items:
+
+* `Roles`: Set the home page, priority, and permissions of a role.
+* `Groups`: Set the home page, priority, and permissions of a group as well as create and delete groups.
+* `Users`: Create and delete users as well as add a group or role to a user and view its permissions.
+* `Artifacts`: View M2 repository artifacts, upload artifacts, view, and download JAR files.
+* `Data Source`: Add, update, or delete data sources and database drivers.
+* `Data Sets`: Create, modify, or delete data sets.
+* `Projects`: View and edit project preferences such as file export properties, space properties, default values, and advanced GAV properties.
+* `Artifact Repository`: Manage artifact repository properties.
+* `Languages`: Set the {CENTRAL} language.
+* `Process Designer`: Set diagram editor properties.
+
+.Prerequisites
+* Installed Red Hat JBoss Enterprise Application Platform 7.1.0. For more information, see {URL_INSTALLING_ON_EAP}[_{INSTALLING_ON_EAP}_].
+* Installed {PRODUCT}. For more information, see the {URL_INSTALLING_DM_ON_PREMISE}[_{INSTALLING_DM_ON_PREMISE}_].
+* {PRODUCT} is running and you can log in to {CENTRAL} with the `admin` role.
+
+// Modules - concepts, procedures, refs, etc.
+include::{enterprise-dir}/admin-and-config/managing-business-central-user-management-con.adoc[leveloffset=+1]
+
+include::{enterprise-dir}/admin-and-config/business-central-settings-creating-new-users-proc.adoc[leveloffset=+2]
+
+include::{enterprise-dir}/admin-and-config/managing-business-central-editing-users-proc.adoc[leveloffset=+2]
+
+include::{enterprise-dir}/admin-and-config/managing-business-central-creating-groups-proc.adoc[leveloffset=+2]
+
+include::{enterprise-dir}/admin-and-config/managing-business-central-editing-groups-proc.adoc[leveloffset=+2]
+
+include::{enterprise-dir}/admin-and-config/managing-business-central-artifacts-proc.adoc[leveloffset=+1]
+
+include::{enterprise-dir}/admin-and-config/managing-business-central-data-sources-con.adoc[leveloffset=+1]
+
+include::{enterprise-dir}/admin-and-config/managing-business-central-adding-database-driver-proc.adoc[leveloffset=+2]
```

Comment:
I'd cover first the driver, as without a driver you can't create a data source

Reply:
Changed as suggested.